test/screenshotter/ss_data.yaml

```diff
@@ -144,6 +144,10 @@ LowerAccent: |
     \end{matrix}
 MathAtom: a\mathrel{\mathop{=}\limits^{\blue ?}}b
 MathAtom2: \mathop{\overline{\mathrm{lim}}}\limits_{x\to\infty}f(x)
+MathChoice: |
+  {\displaystyle\mathchoice{D}{T}{S}{SS}} {\textstyle\mathchoice{D}{T}{S}{SS}} {\scriptstyle \mathchoice{D}{T}{S}{SS}} {\scriptscriptstyle\mathchoice{D}{T}{S}{SS}}
+MathChoice2: |
+  \displaystyle X_{\mathchoice{D}{T}{S}{SS}_{\mathchoice{D}{T}{S}{SS}}}
```

Comment:
Can you merge these two tests (e.g. side-by-side)? I think we try to keep down the number of snapshot images, and these tests render small enough (and they're related) that it's easy to combine them.

Reply:
Sure. I've just combined them into one, and here is the result:

![mathchoice](https://user-images.githubusercontent.com/21860/33126738-f434b0d6-cfc8-11e7-9afe-4df019ed5973.png)